drivers/cc2420/include/cc2420_registers.h

```diff
@@ -36,8 +36,7 @@ extern "C" {
 /** @} */
 
 /**
- * @name    Possible device state change commands
- * @{
+ * @name    device state change commands
```

Comment:
`@name` should always come with an `@{` ... `@}`. If there is no `@{` ... `@}`, use `@brief` instead.

Reply:
In the description of this PR I wrote: there `@name` may or may not open an implicit group (ask the doxygen people why)- (in this case it does not) 
If there is an implicit group it needs to be made explicit since the group counting does not work (on our version of doxygen) for these implicit groups